README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Name
+
+Hello, my name is **Nadiah Wong**! 
+
+
+## Who am I?
+
+I was born on 6 July 1997 in sunny Singapore!
+This is what I look like: ![Japan 2019 during Winter Break](<https://github.com/nadiahwong/hw01/blob/master/me.png>)
```

Comment:
Generally a better approach to embed images on a separate line in the document. Avoids potential formatting issues in the future. Also you don't need to wrap the URL with `<>` inside of the `()`

```
This is what I look like:

![Japan 2019 during Winter Break](https://github.com/nadiahwong/hw01/blob/master/me.png)
```

Reply:
And since the image is in the same repo as the readme, you can even shorten it to `![Japan 2019 during Winter Break](me.png)`